packages/esm-bed-management-app/src/bed-type/bed-type-admin-form.component.tsx

```diff
@@ -149,16 +149,16 @@ const BedTypeAdministrationForm: React.FC<BedAdministrationFormProps> = ({
               />
             )}
           </Stack>
-        </ModalBody>
-        <ModalFooter>
-          <Button onClick={() => onModalChange(false)} kind="secondary">
-            {t('cancel', 'Cancel')}
-          </Button>
-          <Button disabled={!isDirty} type="submit">
-            <span>{t('save', 'Save')}</span>
-          </Button>
-        </ModalFooter>
-      </Form>
+        </Form>
+      </ModalBody>
+      <ModalFooter>
+        <Button onClick={() => onModalChange(false)} kind="secondary">
+          {getCoreTranslation('cancel', 'Cancel')}
+        </Button>
+        <Button disabled={!isDirty} onClick={handleSubmit(onSubmit, onError)}>
```

Comment:
Same here. I imagine we only need this on line 85.

Reply:
Thank you for the feedback, realized it works if I leave this `handleSubmit` implementation here and remove the one on line 85.